lib/chkhash.h

```diff
@@ -0,0 +1,9 @@
+#ifndef _CHKHASH_H
+#define _CHKHASH_H
+
+#include <stdbool.h>
+#include <stddef.h>
```

Comment:
Please include `<config.h>` before all other headers, and separate that one from the others by one blank.